sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -466,13 +467,15 @@ case class AlterTableRenamePartitionCommand(
       oldPartition,
       table.partitionColumnNames,
       table.identifier.quotedString,
-      sparkSession.sessionState.conf.resolver)
+      sparkSession.sessionState.conf.resolver,
+      sparkSession.sessionState.conf.caseSensitiveAnalysis)
 
     val normalizedNewPartition = PartitioningUtils.normalizePartitionSpec(
       newPartition,
       table.partitionColumnNames,
       table.identifier.quotedString,
-      sparkSession.sessionState.conf.resolver)
+      sparkSession.sessionState.conf.resolver,
+      sparkSession.sessionState.conf.caseSensitiveAnalysis)
```

Comment:
seems to me `sparkSession.sessionState.conf.caseSensitiveAnalysis` and `sparkSession.sessionState.conf.resolver` are kind of redundant, can we just use `resolver` to detect duplication?